tests/python/frontend/onnx/test_forward.py

```diff
@@ -3356,15 +3356,19 @@ def verify(ishape, oshape, scales, mode, coord_trans):
 
     # upsampling
     verify([1, 16, 32, 32], [1, 16, 64, 64], [], "nearest", "asymmetric")
+    verify([1, 16, 32, 32], [1, 16, 64, 64], [], "linear", "asymmetric")
+    verify([1, 16, 32, 32], [1, 16, 64, 64], [], "nearest", "align_corners")
     verify([1, 16, 32, 32], [1, 16, 64, 64], [], "linear", "align_corners")
+    verify([1, 16, 32, 32], [1, 16, 64, 64], [], "nearest", "half_pixel")
     verify([1, 16, 32, 32], [1, 16, 64, 64], [], "linear", "half_pixel")
-    # downsampling
```

Comment:
Why remove down-sampling tests?

Reply:
Whoops, I thought they were the same as the upsampling tests (I can read).. I'll add them back